torch/_dynamo/variables/builtin.py

```diff
@@ -1485,6 +1485,9 @@ def _unimplemented():
             if type(left) is not type(right):
                 return ConstantVariable.create(False)
 
+        if isinstance(left, BuiltinVariable) and isinstance(right, BuiltinVariable):
+            return ConstantVariable.create(op(left.fn, right.fn))
+
```

Comment:
What's this for?

Reply:
that's for comparing the tree_spec, where we want to inline call the .__eq__ and there is a field in the spec is `tuple`